Opower integration - Add ConEd

rct opened the pull request:
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
Update Opower integration doc to include ConEd MFA configuration and ORU (opower) - https://github.com/home-assistant/core/pull/97878 


## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/97878 and https://github.com/home-assistant/core/pull/99230 
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4625
- This PR fixes or closes issue: fixes #

Note: ConEd and ORU support is currently merged in opower but is disabled. Merging the MFA PR https://github.com/home-assistant/core/pull/97878 will enable it.  This doc PR should be merged after that PR to document the availability of the two new utilities.

(There is separate PR to add two new virtual integrations: https://github.com/home-assistant/core/pull/99230  This should be merged after https://github.com/home-assistant/core/pull/97878   There is a separate doc PR for the virtual integrations: https://github.com/home-assistant/home-assistant.io/pull/28726)

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

rct committed:
Opower integration - Add ConEd

tronikos commented on source/_integrations/opower.markdown:
```diff
@@ -30,9 +31,35 @@ More than 175 utilities use Opower. Currently only the following ones are suppor
+
+### Exelon subsidiaries
+
+When using Opower with any of the Exelon subsidiaries, such as such as BGE, ComEd, PECO,Pepco, etc., you need to actively disable two-factor authentication. Log onto the website, select **Don't use 2FA** and **Don't ask me again**. If you have already enabled 2FA, disable it.
```
missing space after the comma

tronikos commented on source/_integrations/opower.markdown:
```diff
@@ -17,8 +17,9 @@ ha_integration_type: integration
-More than 175 utilities use Opower. Currently only the following ones are supported:
+More than 175 utilities use Opower. Currently only the following utilities are supported by this integration:
 
+- Consolidated Edison (ConEd)
```
Add "Orange & Rockland Utilities (ORU)" below this, similar to https://github.com/tronikos/opower/blob/main/README.md

rct committed:
Opower docs, Add ORU along wtih ConEd.

Orange & Rockland Utility (ORU) will also be enabled once
the MFA changes for ConEd are merged.

tronikos commented:
Maintainers prefer small PRs. Split this to two separate PRs, each linked to one of the two parent PRs.

rct committed:
Merge branch 'next' into feat-opower-coned-add

rct commented:
> Maintainers prefer small PRs. Split this to two separate PRs, each linked to one of the two parent PRs.

Since the chances are so small and interrelated I thought it would be easier to follow if they were kept together.  

However, as you suggest, I've removed the commits for the new virtual integrations so this PR only covers the Opower page and will add a separate doc PR to accompany the virtual integration code change.
